chart/templates/role-binding.yaml

```diff
@@ -0,0 +1,16 @@
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "loki-rule-operator.fullname" . }}
+  namespace: {{ .Values.lokiConfigmapRulesNamespace }}
+  labels:
+    {{- include "loki-rule-operator.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ include "loki-rule-operator.fullname" . }}
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.serviceAccount.name }}
```

Comment:
Shouldn't this be `{{ include "loki-rule-operator.fullname" . }}` when `.Values.serviceAccount.create` is `false`?